test/fixtures/other/expression-in-inline-tag/output.astro

```diff
@@ -0,0 +1,18 @@
+<span class="class"
+  >{
+    [
```

Comment:
Is this right? I know HTML doesn't have expressions, so what does JSX do with `{[` in this case?

Reply:
JSX doesn't have HTML sensitivity, so it doesn't care at all about those rules. [See this playground example](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAzgBwIZQARgA2266AvADrjGlUX54B8w9ebeA2q+21YdAHMqAGm48+gkWPYSoQkKIbiQ-OVKUyVkhdN5a1OjXtXzFPY9rPnZp3Xhvrz9-baPO4AD0wAnOKQCW0I7sALoAdABWEP5QABR0IACU3AC+qAD0WLj0jAogEJgwgVDoyKDY3t4QAO4AChUIpSjYhNXYAJ6lwiAARt7YYADWcDAAytgAtnAAMjFwyABmLehw3X0Dw2M4YDECyDDeAK6rIHATPXAAJpdX07gCh9gCcABiEN4T2DBFcsgg2IcYBA8gALGATQgAdRB-ngWTAcFGjVh-gAbrD2n8wKQ8jEVt4YLV+gJPotlicIugPKNdoQ4ABFQ4QeBkwgrbo4bz4v49bAXQgAWhgXRAPhiMEh-kuMBByAAHAAGDlVFaQ-qYP4+Pxwbyo+bdACOTPgRIKTX+6AFUDgVyueV8Rv8viJT1JSCWbJOKwm-n2Ry9tIZxvm7vJ3RgfMl0tlSAATOH+v5CLsAMIQCZu07oACseUOKwAKnymh72SBUccAJJQG6wUZgbz+QoAQRroxg7TprJWKRSQA)

Reply:
Ah, I guess I was wondering if the `{[` should hug each other (like JSX), but extrapolating that out to the end tag shows the issue—unless you hug everything to the `>` or `</`, it will introduce leading/trailing whitespace inside of the `span` tag.

This seems like the most reasonable pattern, since the alternative doesn't make sense!